docs/rfc/identity_resolution.md

```diff
@@ -81,20 +81,21 @@ The namespacing scheme could look as follows:
                 |-- heads # <-- code branches owned by $PEER_ID go here
                 |-- rad
                 |   |-- id # <-- points to the identity document history
+                |   |-- refs # <-- signature document
```

Comment:
Nope, not this one -- its the blob which contains the signed refs of the peer. Feel free to conjure a different name if this sounds like `malkovich, malkovich`. `signed_refs` for example.

Reply:
Wait... The jokes has me confused :sweat_smile: It _is_ called `refs` at the moment right? Are you just commenting on the description or the name or both?